apps/web/styles/globals.css

```diff
@@ -1,4 +1,6 @@
 /* @import rules must precede all rules aside from @charset and @layer statements. */
+@import "@react-awesome-query-builder/ui/css/styles.css";
+@import "@react-awesome-query-builder/ui/css/compact_styles.css";
```

Comment:
Might be better to import these two directly in apps/web/styles/raqb.css so that globals.css has single import for raqb